.gitignore

```diff
@@ -6,3 +6,8 @@ node_modules
 vscode-sbt-scala/client/server
 npm-debug.log
 *.vsix
+.classpath
+.project
+.settings
+.cache*
+bin/
```

Comment:
Tool specific rules should go into global gitignore https://help.github.com/articles/ignoring-files/